samples/net/wifi/README.rst

```diff
@@ -0,0 +1,50 @@
+.. _wifi:
```

Comment:
I'd recommend a more unique name, maybe ``wifi_sample``  (all these document labels must be unique across the document set).